cloud/azure/main.tf

```diff
@@ -206,6 +119,55 @@ resource "azurerm_log_analytics_workspace" "civiform_logs" {
   retention_in_days   = 30
 }
 
+output "app_service_default_hostname" {
+  value = "https://${azurerm_app_service.civiform-app.default_site_hostname}"
+}
+
```

Comment:
suggestion (non-blocking): separate file for outputs?

Reply:
done